components/Footer.tsx

```diff
@@ -111,16 +111,18 @@ function Footer() {
     utilityLinks.push({ id: '1', name: t('contact'), slug: plan.contactLink });
   }
 
-  if (plan.externalFeedbackUrl) {
-    utilityLinks.push({
-      id: '2',
-      name: t('give-feedback'),
-      slug: plan.externalFeedbackUrl,
-    });
-  } else if (pathname !== '/feedback') {
-    const url = getFeedbackUrl(pathname);
-    if (url != null) {
-      utilityLinks.push({ id: '2', name: t('give-feedback'), slug: url });
+  if (theme.showFeedbackLink !== false) {
+    if (plan.externalFeedbackUrl) {
+      utilityLinks.push({
+        id: '2',
+        name: t('give-feedback'),
+        slug: plan.externalFeedbackUrl,
+      });
+    } else if (pathname !== '/feedback') {
```

Comment:
I would probably put the `showFeedbackLink` check here. If someone has set `externalFeedbackUrl` in the admin we can safely assume that that should be an override. 

Would something like `(pathname !== '/feedback' && theme.showFeedbackLink !== false)` work?

Reply:
Thanks @terotik! fixed the logic